bin/list-all

```diff
@@ -14,5 +14,5 @@ function sort_versions() {
 }
 
 # Fetch all tag names, and get only second column. Then remove all unnecesary characters.
-versions=$(eval $cmd | grep -oE "tag_name\": \".{1,15}\"," | sed 's/tag_name\": \"v//;s/\",//' | sort_versions)
+versions=$(eval $cmd | grep -oE "tag_name\":\".{1,15}\"," | sed 's/tag_name\":\"v//;s/\",//' | sort_versions)
```

Comment:
Can we use `\s*` instead of just removing the space? (Or whatever the equivalent of `grep`s space symbol is)

Reply:
I just tested this and it works fine, I have now amended the commit to use \s*.